### DISCUSSION TITLE
About a easier version of this repo

### DISCUSSION THREAD

**Discussion by ilyasyldrm0** [schwabe/ics-openvpn#1724] [General]
Hi there, i'm software engineer. I found this repo very helpful about content. It's very helpful but i expect a little easier understanding for the peoples who doesnt know about this type of project. It could be a little description about how to compile this project what should we prepare before it and explaining about how can u use it on another project etc.. Because you know, a lot of android developers dont know how to use  JNI(Java Native Interface) or C++. Thanks for your efforts, this is just comment. @schwabe

**Comment by schwabe** [comment]
I am not sure what you are looking for. This is a security and OpenVPN software. It is mainly in C/C++ since that is the language that OpenVPN is written in. The readme already has a README. And even if this might sounds hostile, it is okay for me that you need to be a more experienced developer to be able to contribute to this project as working with security related software should be done by more expirenced developer/developers who take the time and effort to understand these thngs.
